README.md

```diff
@@ -36,10 +45,14 @@ module ([`/example`](examples/)),
 ```
 
 ## Handling Blockstack sessions
-The current implementation of the Blockstack Android SDK uses the j2v8 engine and blockstack.js. 
+Blockstack sessions are stored in the SessionStore. The default implementation of SessionStore uses 
+the default shared preferences. If an apps needs to use two `BlockstackSession`s then they should 
+use the same session store or synchronize the state regularly.  
+
 
 ### Redirect with app links
-The example applications and tutorial uses a custom url scheme to handle the redirect from the sign-in process. In production, the redirect should be handled by [app links](https://developer.android.com/studio/write/app-link-indexing) such that no other apps could hijack the custom url scheme. (There is no security risk, it is just a bad user experience if an app chooser pops up and the user has to choose how to finish the sign-in.)
+The example applications and tutorial uses a custom url scheme to handle the redirect from the sign-in process. 
```

Comment:
The example applications and tutorial uses a custom url scheme to handle the redirect from the sign-in process. While suitable for samples, do not use this scheme in a production app. 

In production, handle the redirect with [app links](https://developer.android.com/studio/write/app-link-indexing) such that no other apps could hijack the custom url scheme. Hijacking in this manner is not a security risk; It is simply a bad user experience if an app chooser pops up and the user has to choose how to finish the sign-in.